### PR TITLE
Add mono fonts also well as some references

### PR DESCRIPTION
**Pull request by mlcdf**
Hi,

In most of my projects, I find myself having to go check one of your website in order to steal _that_ font-family declaration:

``` css
font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, ubuntu, roboto, noto, segoe ui, helvetica, helvetica neue, arial, sans-serif;
```

 And so, I really love the idea of this tiny lib. No more copy-pasting.

But since I also use monospace fonts, I thought it would be great to add them to this lib. Let me know what you think. :wink:

Max

### CODE REVIEW

**Comment by ungoldman** [comment]
Nice, was just about to suggest the same! Heres' the list I compiled for reference:

``` css
.system-monospace {
  font-family:  menlo, /* mac */
                inconsolata, consolas, /* windows */
                'fira mono', /* firefox os */
                'noto mono', 'droid sans mono', /* android */
                'dejavu sans mono', 'liberation mono', 'ubuntu mono', /* linux */
                monaco, 'courier new', /* older systems */
                monospace; /* fallback */
}
```

**Comment by ungoldman** [comment]
Would be great to get a `.system-serif` list too to have the whole set available

**Comment by mlcdf** [comment]
Something like this ?

``` css
.system-serif { 
  font-family: Georgia, 'Times New Roman', /* MacOS, Windows and some Linux*/
              'Droid Serif', /* Old Google stuff */
              'DejaVu Serif', 'Liberation Serif', /* Linux */
              serif; /* Everything else */
}
```

**Comment by ungoldman** [comment]
I definitely wouldn't lead with Georgia and Times New Roman. [Charter](https://en.wikipedia.org/wiki/Bitstream_Charter) is a really nice serif that is available on a lot of systems. I was thinking the serif set should be as modern and up to date as the sans-serif, using the best each system has to offer and only relying on older generics as fallbacks.

**Comment by mlcdf** [comment]
Yeah sure, you're right. I'll change the order and put Georgia and Times New Roman as fallback fonts.
And, since I don't really know any good serif fonts for macOS or Windows, I'm open to recommendations. :)

File: css/system-fonts.css [hunk 1]
```diff
@@ -15,13 +15,24 @@
                'franklin gothic medium', 'century gothic', /* Windows stuff */
                'Liberation Sans', /* Linux */
                sans-serif; /* Everything else */ }
+.system-mono { font-family: Menlo, Monaco, /* MacOS */
+              'Segoe UI Mono, Lucida Console', /* Windows */
```
Comment (mvasilkov) [inline css/system-fonts.css:19]: Segoe UI Mono didn't come with any version of Windows; it was a part of some XNA addon for Visual Studio.

**Pull request closed without merging**